doc/cookbook/engine.rst

```diff
@@ -0,0 +1,84 @@
+Engines
+=======
+
+Ploomber currently supports 2 engines for running notebooks:
+
+- Ploomber Engine
+- Papermill Engine
```

Comment:
```suggestion
- Papermill (default)
```

let's add a link to papermill's here

also put papermill as the first element since it's the default